### DISCUSSION TITLE
code needs updated to use new webui repo

### DISCUSSION THREAD

**Discussion by nagaisa** [acheong08/Diffusion-ColabUI#74] [Announcements]
AUTOMATIC1111 made a post on the stable diffusein sub about how his account got suspended and the repo is not available (at least for now) here's the [post ](https://www.reddit.com/r/StableDiffusion/comments/103vsm2/webuis_new_home/) this is the new repo https://gitgud.io/AUTOMATIC1111/stable-diffusion-webui/

**Comment by acheong08** [comment]
The UI uses my fork of his repo which means it should keep working for now. I just need to manually pull the changes into github

**Reply by acheong08** [reply]
I see there are a few changes I need to make to use some of the features. Will try to pull them somehow

**Reply by acheong08** [reply]
https://github.com/acheong08/stable-diffusion-webui/commit/cca6e5212ddd77be7efdce2c27e475d3af436d85

**Comment by acheong08** [comment]
I see from archive.org that Automatic1111 has some extremely questionable repos that were taken down

**Reply by acheong08** [reply]
There are allegations floating around that he is racist. I personally disagree that mods that mention a color are racist but I see how it can be viewed as such these days. (For context, I am not white and do not support racist ideologies. I am simply stating that not everything is racist)
